content/en/docs/project-user-guide/custom-application-monitoring/get-started/monitor-sample-web.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Monitor Sample Web"
+keywords: 'monitoring, prometheus, prometheus operator'
+description: 'Monitor Sample Web'
+
+linkTitle: "Monitor Sample Web"
+weight: 2120
+---
+
+This section walks you through monitoring a sample web application. The application is instrumented with Prometheus Golang client in its code. Therefore, it can expose metrics directly without the help of exporters.
+
+## Prerequisites
+
+- Please make sure you enable the OpenPitrix system.
+- You need to create a workspace, a project, and a user account for this tutorial. This account needs to be a platform regular user and to be invited as a self provisioner. In this tutorial, we will work with the workspace `my-workspace` and the project `demo`. We will log in as `project-self-provisioner` and create the project `demo` beforehand.
+- You know Helm Chart.
+- You know [PromQL](https://prometheus.io/docs/prometheus/latest/querying/examples/).
+
+## Hands-on Lab
+
+### Step 1: Prepare Sample Web Application Image
+
+First, prepare the sample web application image, The sample web application exposes a user-defined metric called `myapp_processed_ops_total`. It is a counter type metric that counts the number of operations that have been processed by far. The counter auto increases by one every 2 seconds. 
```

Comment:
prepare the sample web application image,
->
prepare the sample web application image.

Reply:
done